recipe/meta.yaml

```diff
@@ -1,23 +1,23 @@
 {% set name = "abacus" %}
-{% set version = "3.9.0.17" %}
+{% set version = "3.10.1" %}
 {% set proc_type = "cuda" if cuda_compiler_version != "None" else "cpu" %}
 
+{% set enable_ml = (environ.get('ABACUS_ENABLE_MLALGO', '0') | lower in ['1', 'true', 'yes']) %}
+{% set ml_flag = '-DENABLE_MLALGO=1' if version.startswith('3.9.0') else '-DENABLE_DEEPKS=1' %}
```

Comment:
@caic99 do we need to keep them? I don't find the uses of them.

Reply:
I think there's no need to keep them as per my earlier comment.